.gitignore

```diff
@@ -15,3 +15,7 @@ testcase2/doc/build/
 parser/*.fmu
 parser/*.mo
 parser/*.json
+
+xx.ipynb
+connect_boptest.py
```

Comment:
I don't see a connect_boptest.py and don't think this should be written in the .gitignore.